### PR TITLE
fix: brought context menu forward in editor

### LINKED ISSUE #620: Monaco tooltips should be in foreground.
This is a regression coming from a recent refactor I did. 

Screen Shot 2020-02-19 at 10 04 45

### PR DESCRIPTION
**Pull request by adrijshikhar**
fix #620

### CODE REVIEW

File: src/components/input-panel/index.css [hunk 1]
```diff
@@ -7,5 +7,8 @@
 
 .full-height-wrapper > .full-height-wrapper {
   flex: 1;
-  overflow: hidden;
+}
+
+.full-height-wrapper .react-monaco-editor-container {
```
[the remark this answers is not in the document]
Comment (domoritz) [reply src/components/input-panel/index.css]: Thanks for helping with reviews.

**Comment by adrijshikhar** [comment]
Screenshot_20200220_225116

**Comment by ashu8912** [comment]
No But The scroll from above editor is missing

**Comment by adrijshikhar** [comment]
ok let me fix it

**Comment by adrijshikhar** [comment]
When the page is reloaded with the compiled tab opened, the scroll bar is visible and scrolling is possible but as soon as you close the complied tab and re open it, the scroll bar disappears. has anyone faced this kind of issue before?

**Comment by domoritz** [comment]
Also try resizing the editor panels. 

Btw, I don't want to use references in React since they are deprecated.

**Comment by adrijshikhar** [comment]
https://github.com/vega/editor/blob/9474b5f977a7a565c305149b9130eb59c6db89d7/src/components/input-panel/spec-editor/renderer.tsx#L249-L255

**Comment by domoritz** [comment]
It's hard to answer that question. Feel free to make the changes necessary to have a) a simple implementation and b) don't use deprecated features.

**Comment by domoritz** [comment]
Fixed in master

**Pull request closed without merging**